src/libstd/collections/hash/table.rs

```diff
@@ -126,9 +126,10 @@ unsafe impl<K: Send, V: Send> Send for RawTable<K, V> {}
 unsafe impl<K: Sync, V: Sync> Sync for RawTable<K, V> {}
 
 pub struct RawBucket<K, V> {
-    hash: *mut HashUint,
+    _hash: *mut HashUint,
```

Comment:
It's for disambiguation. I think we can come up with a better name instead, moving the _ to the end hurts the eye. Like `hash_start`, f`hash_base, ' first_hash`?